### PR TITLE
Various improvements related to the new X-ray transform implemementation.

### PR DESCRIPTION
**Pull request by bwohlberg**
Various improvements, mostly related to the new X-ray transform implemementation, including renaming of tomographic projector classes for consistency.

### CODE REVIEW

File: examples/scripts/ct_tv_admm.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+TV-Regularized Sparse-View CT Reconstruction (Integrated Projector)
```
Comment (Michael-T-McCann) [inline examples/scripts/ct_tv_admm.py:8]: Does the notebook need to be (re)generated for this?

File: scico/linop/xray/_xray.py [hunk 2]
```diff
@@ -20,22 +19,25 @@
...
+class XRayTransform(LinearOperator):
+    """X-ray transform operator.
 
-    Wraps an X-ray projector object in a SCICO
-    :class:`LinearOperator`.
+    Wrap an X-ray projector object in a SCICO :class:`LinearOperator`.
+    **Warning:**  Note that the only X-ray projector object currently
+    supported, :class:`.Parallel2dProjector`, is not a very accurate
+    approximation of the integral transform representing real projection
+    imaging, and is therefore not suitable for real imaging applications.
```
Comment (Michael-T-McCann) [inline scico/linop/xray/_xray.py:32]: "may not be suitable"? not trying to defend its accuracy, but we do we know it isn't suitable?

File: scico/optimize/_admmaux.py [hunk 3]
```diff
@@ -222,7 +222,6 @@ def internal_init(self, admm: soa.ADMM):
             # hessian = A.T @ W @ A; W may be identity
             lhs_op += admm.f.hessian
 
-        lhs_op.jit()
```
Comment (Michael-T-McCann) [inline scico/optimize/_admmaux.py:225]: Just checking, is this intentional?

File: scico/test/test_fixture.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+@pytest.fixture(autouse=True)
+def run_before_and_after_tests(tmpdir):
+    """Fixture to execute asserts before and after a test is run"""
+    # Setup: fill with any logic you want
+    print("Before: ", jax.config.read("jax_enable_x64"))
+
+    yield  # this is where the testing happens
+
+    # Teardown : fill with any logic you want
+    print("After: ", jax.config.read("jax_enable_x64"))
```
Comment (Michael-T-McCann) [inline scico/test/test_fixture.py:15]: Should this be here? Looks like debugging code (though might be useful to have).

**Review by Michael-T-McCann — APPROVED** [review]
